src/pages/PageNotFound404.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link, useLocation } from "react-router-dom";
+
+export default function PageNotFound404() {
```

Comment:
can we rename this to PageNotFoundView. The 404 seems unnecessary and seems to be messing up naming conventions through out the application